apps/fabric-website-resources/package.json

```diff
@@ -67,7 +69,12 @@
     "@uifabric/theme-samples": ">=0.1.4 <1.0.0",
     "@uifabric/utilities": ">=6.35.1 <7.0.0",
     "@uifabric/variants": ">=6.14.0 <7.0.0",
+    "expect-puppeteer": "4.1.0",
+    "jest-environment-node": "24.5.0",
+    "jest-environment-puppeteer": "^4.1.0",
```

Comment:
Can you make these ~ dependencies instead of ^ ? (I've been meaning to go through and lock down all the existing deps that way)

Reply:
Agree - the ^ has bit us in the past (recent past :D)